### ISSUE TITLE
Schematica Blocker Doesnt Work. I tried multiple versions of the mod.

### ISSUE THREAD

**Issue by PineappleCactus** [dvargas135/eZProtector#46]
The Schematica Feature to block Printer mode does not work. Can someone please assist me?

**Comment by CloudeLecaw** [comment]
No, if it does not work there  will be no fix anymore.  Also there need to be no assist, the feature work or not (what would means it must be fixed) but there will be no development anymore...thats the devs current statment..

**Comment by DoNotSpamPls** [comment]
It should be working in the newest eZProtector version now 😃
